voltdb-hadoop/src/main/java/org/voltdb/hadoop/VoltConfiguration.java

```diff
@@ -82,6 +85,17 @@
     /** Load destination table name */
     public static final String TABLENAME_PROP = "mapred.voltdb.table.name";
 
+    /** VoltDB client timeout */
+    public static final String CLIENT_TIMEOUT_PROP = "mapred.voltdb.client.timeout";
+    /** Time out default */
+    public static final long TIMEOUT_DFLT =  2 * 60 * 1000;
+
+    /**max number of errors allowed  */
+    public static final String BULKLOADER_MAX_ERRORS_PROP="mapred.voltdb.bulkloader.max.errors";
+
+    /**Bulkloader in upsert mode  */
+    public static final String BULKLOADER_UPSERT_PROP="mapred.voltdb.bulkloader.upsert";
```

Comment:
You should remove this property and we can deal with it in new ticket when we fig out whats merge semantics are.